scripts/policy/misc/capture-loss.zeek

```diff
@@ -65,13 +78,17 @@ event CaptureLoss::take_measurement(last_ts: time, last_acks: count, last_gaps:
 	local info: Info = [$ts=now,
 	                    $ts_delta=now-last_ts,
 	                    $peer=peer_description,
-	                    $acks=acks, $gaps=gaps, 
+	                    $acks=acks, $gaps=gaps,
 	                    $percent_lost=pct_lost];
-	
+
 	if ( pct_lost >= too_much_loss*100 )
-		NOTICE([$note=Too_Much_Loss, 
+		NOTICE([$note=Too_Much_Loss,
 		        $msg=fmt("The capture loss script detected an estimated loss rate above %.3f%%", pct_lost)]);
-	
+
+	if ( acks < minimum_acks )
+		NOTICE([$note=Too_Little_Traffic,
+		        $msg=fmt("The worker only observed %d ACKs and was expecting at least %d.", acks, minimum_acks)]);
```

Comment:
```suggestion
		        $msg=fmt("Only observed %d ACKs and was expecting at least %d.", acks, minimum_acks)]);
```

This script is cluster-agnostic, so maybe worth avoiding terminology that's more frequently used in context of clusters.

Reply:
Yep, makes sense.

Should we also change that to "%d TCP ACKs" to highlight that if you're not looking at TCP traffic, this notice can be ignored?

Reply:
> Should we also change that to "%d TCP ACKs"

Sure.